bigtable/google/cloud/bigtable/row.py

```diff
@@ -313,20 +311,20 @@ def set_cell(self, column_family_id, column, value, timestamp=None):
                                  Must be of the form
                                  ``[_a-zA-Z0-9][-_.a-zA-Z0-9]*``.
 
-        :type column: bytes
+        :type column: str
         :param column: The column within the column family where the cell
                        is located.
 
-        :type value: bytes or :class:`int`
+        :type value: str or :class:`int`
```

Comment:
Cell values are likewise bytes, not text, in the [backend docs](https://cloud.google.com/bigtable/docs/reference/data/rpc/google.bigtable.v2#google.bigtable.v2.Cell).